app/src/full/java/io/homeassistant/companion/android/sensors/LocationSensorManager.kt

```diff
@@ -625,7 +625,13 @@ class LocationSensorManager : BroadcastReceiver(), SensorManager {
             .setInitialTrigger(GeofencingRequest.INITIAL_TRIGGER_ENTER)
 
         // TODO cache the zones on device so we don't need to reach out each time
-        val configuredZones = integrationUseCase.getZones()
+        var configuredZones: Array<Entity<ZoneAttributes>> = emptyArray()
+        try {
+            configuredZones = integrationUseCase.getZones()
+        } catch (e: Exception) {
+            Log.e(TAG, "Error receiving zones from Home Assistant")
```

Comment:
```suggestion
            Log.e(TAG, "Error receiving zones from Home Assistant", e)
```

lets make sure to print the actual error

Reply:
Yes you are right, even though the exception is empty when thrown. Better to have, than not to have :)